`asyncapi` binding triggers zilla crash when used with `catalog::apicurio`

<!--
  - Please search to see if an issue already exists for the issue you encountered. Duplicate issues will be closed. [Current GitHub Issues](https://github.com/aklivity/zilla/issues)
  - Please make sure your Zilla version is up to date. [Latest Zilla Release](https://github.com/aklivity/zilla/releases/latest)
-->

**Describe the bug**
Using `catalog::apicurio` with `bindings:asyncapi` triggers `zilla` crash with mentioned exception:

```
java.util.concurrent.CompletionException: java.lang.IllegalAccessError: class io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper (in module io.aklivity.zilla.runtime.binding.asyncapi) cannot access class io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig (in module io.aklivity.zilla.runtime.catalog.apicurio) because module io.aklivity.zilla.runtime.catalog.apicurio does not export io.aklivity.zilla.runtime.catalog.apicurio.internal.config to module io.aklivity.zilla.runtime.binding.asyncapi
        at java.base/java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:413)
        ...
        at io.aklivity.zilla.runtime.command@0.9.90/io.aklivity.zilla.runtime.command.internal.ZillaMain.main(ZillaMain.java:34)
Caused by: java.lang.IllegalAccessError: class io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper (in module io.aklivity.zilla.runtime.binding.asyncapi) cannot access class io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig (in module io.aklivity.zilla.runtime.catalog.apicurio) because module io.aklivity.zilla.runtime.catalog.apicurio does not export io.aklivity.zilla.runtime.catalog.apicurio.internal.config to module io.aklivity.zilla.runtime.binding.asyncapi
        at io.aklivity.zilla.runtime.binding.asyncapi@0.9.90/io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper.injectApicurioRegistry(AsyncapiCompositeGenerator.java:310)
        ..
        at java.base/java.lang.Thread.run(Thread.java:1570)
java.lang.IllegalAccessError: class io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper (in module io.aklivity.zilla.runtime.binding.asyncapi) cannot access class io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig (in module io.aklivity.zilla.runtime.catalog.apicurio) because module io.aklivity.zilla.runtime.catalog.apicurio does not export io.aklivity.zilla.runtime.catalog.apicurio.internal.config to module io.aklivity.zilla.runtime.binding.asyncapi
error
io.aklivity.zilla.runtime.engine.config.ConfigException: Engine configuration failed
        at io.aklivity.zilla.runtime.engine@0.9.90/io.aklivity.zilla.runtime.engine.internal.registry.EngineManager.onPathChanged(EngineManager.java:207)
        ...
        at io.aklivity.zilla.runtime.command@0.9.90/io.aklivity.zilla.runtime.command.internal.ZillaMain.main(ZillaMain.java:34)
Caused by: java.util.concurrent.CompletionException: java.lang.IllegalAccessError: class io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper (in module io.aklivity.zilla.runtime.binding.asyncapi) cannot access class io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig (in module io.aklivity.zilla.runtime.catalog.apicurio) because module io.aklivity.zilla.runtime.catalog.apicurio does not export io.aklivity.zilla.runtime.catalog.apicurio.internal.config to module io.aklivity.zilla.runtime.binding.asyncapi
        at java.base/java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:413)
        ...
        at io.aklivity.zilla.runtime.engine@0.9.90/io.aklivity.zilla.runtime.engine.internal.registry.EngineManager.onPathChanged(EngineManager.java:183)
        ... 8 more
Caused by: java.lang.IllegalAccessError: class io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper (in module io.aklivity.zilla.runtime.binding.asyncapi) cannot access class io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig (in module io.aklivity.zilla.runtime.catalog.apicurio) because module io.aklivity.zilla.runtime.catalog.apicurio does not export io.aklivity.zilla.runtime.catalog.apicurio.internal.config to module io.aklivity.zilla.runtime.binding.asyncapi
        at io.aklivity.zilla.runtime.binding.asyncapi@0.9.90/io.aklivity.zilla.runtime.binding.asyncapi.internal.config.composite.AsyncapiCompositeGenerator$NamespaceHelper$CatalogsHelper.injectApicurioRegistry(AsyncapiCompositeGenerator.java:310)
        ...
        at java.base/java.lang.Thread.run(Thread.java:1570)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Start zilla with `type: "apicurio"` catalog with `type: "asyncapi"` bindings.
2. See above mentioned error

**Expected behavior**
Zilla should be able to handle this config without an issue.

--- a/runtime/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/config/composite/AsyncapiCompositeGenerator.java
+++ b/runtime/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/config/composite/AsyncapiCompositeGenerator.java
@@ -49,7 +49,7 @@ import io.aklivity.zilla.runtime.binding.asyncapi.internal.view.AsyncapiSchemaIt
 import io.aklivity.zilla.runtime.binding.asyncapi.internal.view.AsyncapiSchemaView;
 import io.aklivity.zilla.runtime.binding.asyncapi.internal.view.AsyncapiServerView;
 import io.aklivity.zilla.runtime.binding.asyncapi.internal.view.AsyncapiView;
-import io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig;
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
 import io.aklivity.zilla.runtime.catalog.inline.config.InlineOptionsConfig;
 import io.aklivity.zilla.runtime.catalog.inline.config.InlineOptionsConfigBuilder;
 import io.aklivity.zilla.runtime.catalog.karapace.config.KarapaceOptionsConfig;

--- a/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/config/ApicurioOptionsConfig.java
+++ b/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/config/ApicurioOptionsConfig.java
@@ -12,7 +12,7 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package io.aklivity.zilla.runtime.catalog.apicurio.internal.config;
+package io.aklivity.zilla.runtime.catalog.apicurio.config;
 
 import java.time.Duration;
 import java.util.function.Function;

--- a/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/config/ApicurioOptionsConfigBuilder.java
+++ b/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/config/ApicurioOptionsConfigBuilder.java
@@ -12,7 +12,7 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package io.aklivity.zilla.runtime.catalog.apicurio.internal.config;
+package io.aklivity.zilla.runtime.catalog.apicurio.config;
 
 import java.time.Duration;
 import java.util.function.Function;

--- a/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogContext.java
+++ b/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogContext.java
@@ -16,7 +16,7 @@ package io.aklivity.zilla.runtime.catalog.apicurio.internal;
 
 import java.util.concurrent.ConcurrentMap;
 
-import io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig;
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
 import io.aklivity.zilla.runtime.engine.EngineContext;
 import io.aklivity.zilla.runtime.engine.catalog.CatalogContext;
 import io.aklivity.zilla.runtime.engine.catalog.CatalogHandler;

--- a/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogHandler.java
+++ b/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogHandler.java
@@ -14,9 +14,9 @@
  */
 package io.aklivity.zilla.runtime.catalog.apicurio.internal;
 
+import static io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfigBuilder.CONTENT_ID;
+import static io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfigBuilder.LEGACY_ID_ENCODING;
 import static io.aklivity.zilla.runtime.catalog.apicurio.internal.CachedArtifactId.IN_PROGRESS;
-import static io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfigBuilder.CONTENT_ID;
-import static io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfigBuilder.LEGACY_ID_ENCODING;
 import static org.agrona.BitUtil.SIZE_OF_BYTE;
 import static org.agrona.BitUtil.SIZE_OF_INT;
 import static org.agrona.BitUtil.SIZE_OF_LONG;
@@ -42,7 +42,7 @@ import org.agrona.DirectBuffer;
 import org.agrona.collections.Int2ObjectCache;
 import org.agrona.concurrent.UnsafeBuffer;
 
-import io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig;
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
 import io.aklivity.zilla.runtime.catalog.apicurio.internal.types.ApicurioPrefixFW;
 import io.aklivity.zilla.runtime.engine.EngineContext;
 import io.aklivity.zilla.runtime.engine.catalog.CatalogHandler;

--- a/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/config/ApicurioOptionsConfigAdapter.java
+++ b/runtime/catalog-apicurio/src/main/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/config/ApicurioOptionsConfigAdapter.java
@@ -21,6 +21,8 @@ import jakarta.json.JsonObject;
 import jakarta.json.JsonObjectBuilder;
 import jakarta.json.bind.adapter.JsonbAdapter;
 
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfigBuilder;
 import io.aklivity.zilla.runtime.engine.config.OptionsConfig;
 import io.aklivity.zilla.runtime.engine.config.OptionsConfigAdapterSpi;
 

--- a/runtime/catalog-apicurio/src/main/moditect/module-info.java
+++ b/runtime/catalog-apicurio/src/main/moditect/module-info.java
@@ -17,6 +17,8 @@ module io.aklivity.zilla.runtime.catalog.apicurio
     requires java.net.http;
     requires io.aklivity.zilla.runtime.engine;
 
+    exports io.aklivity.zilla.runtime.catalog.apicurio.internal.config;
+
     provides io.aklivity.zilla.runtime.engine.catalog.CatalogFactorySpi
         with io.aklivity.zilla.runtime.catalog.apicurio.internal.ApicurioCatalogFactorySpi;
 

--- a/runtime/catalog-apicurio/src/main/moditect/module-info.java
+++ b/runtime/catalog-apicurio/src/main/moditect/module-info.java
@@ -17,7 +17,7 @@ module io.aklivity.zilla.runtime.catalog.apicurio
     requires java.net.http;
     requires io.aklivity.zilla.runtime.engine;
 
-    exports io.aklivity.zilla.runtime.catalog.apicurio.internal.config;
+    exports io.aklivity.zilla.runtime.catalog.apicurio.config;
 
     provides io.aklivity.zilla.runtime.engine.catalog.CatalogFactorySpi
         with io.aklivity.zilla.runtime.catalog.apicurio.internal.ApicurioCatalogFactorySpi;

--- a/runtime/catalog-apicurio/src/test/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogIT.java
+++ b/runtime/catalog-apicurio/src/test/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/ApicurioCatalogIT.java
@@ -32,7 +32,7 @@ import org.junit.rules.Timeout;
 
 import io.aklivity.k3po.runtime.junit.annotation.Specification;
 import io.aklivity.k3po.runtime.junit.rules.K3poRule;
-import io.aklivity.zilla.runtime.catalog.apicurio.internal.config.ApicurioOptionsConfig;
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
 import io.aklivity.zilla.runtime.engine.EngineContext;
 import io.aklivity.zilla.runtime.engine.catalog.CatalogHandler;
 import io.aklivity.zilla.runtime.engine.model.function.ValueConsumer;

--- a/runtime/catalog-apicurio/src/test/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/config/ApicurioOptionsConfigAdapterTest.java
+++ b/runtime/catalog-apicurio/src/test/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/config/ApicurioOptionsConfigAdapterTest.java
@@ -28,6 +28,8 @@ import jakarta.json.bind.JsonbConfig;
 import org.junit.Before;
 import org.junit.Test;
 
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
+
 public class ApicurioOptionsConfigAdapterTest
 {
     private Jsonb jsonb;

